tools/obscuroscan_v3/frontend/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:18-buster as runner
+# setup container data structure
+RUN mkdir -p /home/obscuro/go-obscuro/tools/obscuroscan_v3/
+COPY ./tools/obscuroscan_v3/frontend /home/obscuro/go-obscuro/tools/obscuroscan_v3/frontend
+
+WORKDIR /home/obscuro/go-obscuro/tools/obscuroscan_v3/frontend
+RUN npm install && npm install http-server -g
```

Probably no need for the http server if next has a a prod server itself
```suggestion
RUN npm install
```